components/nordigen/sources/new-transaction/new-transaction.mjs

```diff
@@ -0,0 +1,165 @@
+import hash from "object-hash";
+import nordigen from "../../nordigen.app.mjs";
+import { axios } from "@pipedream/platform";
+
+// TODO: Add description docs
+export default {
+  key: "nordigen-new-transation",
+  name: "New transaction",
+  description: "Emit new event when a transaction occurs",
+  version: "0.0.1",
+  type: "source",
+  props: {
+    db: "$.service.db",
+    nordigen,
+    timer: {
+      type: "$.interface.timer",
+      label: "Timer",
+      description: "When should the source check for a new event",
+      default: {
+        intervalSeconds: 60 * 60 * 24,
+      },
+    },
+    country_code: {
+      propDefinition: [
+        nordigen,
+        "country_code",
+      ],
+    },
+    institution_id: {
+      propDefinition: [
+        nordigen,
+        "institution_id",
+      ],
+    },
+    access_valid_for_days: {
+      type: "integer",
+      label: "Validity days",
+      description: "Number of days the user agreement will be valid.",
+    },
+    max_historical_days: {
+      type: "integer",
+      label: "Maximum historical days",
+      description: "Number of days the user agreement will grand acess to when listing transactions.",
+    },
+  },
+  methods: {
+    _getRequisitionId() {
+      return this.db.get("requisitionId")
+    },
+    _setRequisitionId(data) {
+      this.db.set("requisitionId", data)
+    },
+    _getPreviousTransactions() {
+      return this.db.get("previousTransactions") || [];
+    },
+    _setPreviousTransactions(data) {
+      this.db.set("previousTransactions", data);
+    },
+    _emitEvent(data) {
+      this.$emit(data, {
+        // TODO: Summary and id???
```

Comment:
About this, please add a property called `id` with a unique identifier.  And `summary` with a short message :)

Reply:
[Doc](https://pipedream.com/docs/components/quickstart/nodejs/sources/#add-an-optional-summary-for-each-emitted-event)